package.json

```diff
@@ -98,6 +98,7 @@
   },
   "dependencies": {
     "@aws-crypto/sha256-js": "^2.0.1",
+    "@aws-sdk/client-chime-sdk-messaging": "^3.0.0",
```

Comment:
Should we make this a peer dependency? So if builder is using AWS SDK v2 they do not have to install `@aws-sdk/client-chime-sdk-messaging` if they are using v3, they can install this dependency on their own? @ltrung any discussion on this?

Reply:
Actually we use it nvm thats invalid comment ^